files/base.rb

```diff
@@ -101,7 +105,7 @@ def full_description
 Occurrences:  #{@event['occurrences']}
 
 Team: #{team_name}
-Host: #{@event['client']['name']}
+Host: #{client_display_name}
```

Comment:
I see, yes, I agree it's a good idea. I will add Client Name to the output of full_description.